subsys/dm/timeslot_queue.c

```diff
@@ -98,7 +98,6 @@ int timeslot_queue_append(struct dm_request *req, uint32_t start_ref_tick)
 	}
 
 	item->timeslot_req.start_time = start_time;
-	req->access_address++;
```

Comment:
maybe I'm missing something here, but where are you creating a new seed for a follow-up ranging? I think the seed should be different for each ranging.